tools/rust_api/src/lib.rs

```diff
@@ -30,6 +30,16 @@
 //! - `KUZU_SHARED`: If set, link dynamically instead of statically
 //! - `KUZU_INCLUDE_DIR`: Directory of kuzu's headers
 //! - `KUZU_LIBRARY_DIR`: Directory containing kuzu's pre-built libraries.
+//!
+//! ## Using Extensions
+//!
+//! If you want to use [extensions](https://docs.kuzudb.com/extensions/) in binaries using this
+//! library, you will need to add the following to your build.rs (or create one) so that the binary
+//! produced acts like a library that the extension can link with. Not doing this will produce
+//! undefined symbol errors when the extension is loaded:
+//! ```ignore
+//! println!("cargo:rustc-link-arg=-rdynamic");
```

Comment:
Does this solve the problem that using `export KUZU_SHARED=1` causes, namely that `cargo install` is not seamless?

Reply:
Yes, this will let the extensions work as normal and there wouldn't be anything a user would have to do. The developers would just have to add that line to their build script.